index.html

```diff
@@ -8,18 +8,28 @@
   <link rel="stylesheet" href="style.css">
 </head>
 <body>
-  <header class="header-background">
-   <nav>
-     <div  class="top-header display-flex">
-      <a class="welcome identical-to" href="#">Welcome</a>
-      <span class="identical-to">&#8801;</span>
-     </div>     
+  <header class="header-background">    
+   <nav>      
       <ul class="desktop-nav">       
         <li class="desktop-nav-list"><a class="text-color" href="#">Portfolio</a></li>
         <li class="desktop-nav-list"><a class="text-color" href="#">About</a></li>
         <li class="desktop-nav-list"><a class="text-color" href="#">Contact</a></li>        
     </ul>
-  </nav>
```

Comment:
- [ ] I really like the way that you implement the closing menu but some people won't how to do it so please try to add the close button from the Figma design 💯

Reply:
fixed